app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,56 @@
+class CommentsController < ApplicationController
+    def index
+        @post = Post.find(params[:post_id])
+        @comments = @post.comments.all
+        redirect_to post_path(@post)
```

Comment:
So when somebody wants to view all the posts, you want to direct them to a specific post page (i.e., `show` instead of `index`)